analyzers/src/ITs.JsonParser/Reports/InputReport.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.IO;
+using System.Text.Json;
+using ITs.JsonParser.Json;
+
+namespace ITs.JsonParser.Reports;
+
+/// <summary>
+/// SARIF report generated during the build process.
+/// </summary>
+internal class InputReport
+{
+    public string Project { get; init; }
+    public string Assembly { get; init; }
+    public string TFM { get; init; }
```

Comment:
As raised by SC. Same on OutpuReport
```suggestion
    public string Tfm { get; init; }
```

Reply:
I'll do it, but I would prefer `TFM`, as it's more obvious it's an acronym and you almost always see it in that form.

Reply:
MS Conding style requires 3+ letters acronyms to be in Aaa form. Any abbreviation like SQL is also in written as Sql in classes.